application_form/api/views.py

```diff
@@ -1,22 +1,49 @@
-from rest_framework.permissions import IsAuthenticated
+from django.views.decorators.http import require_http_methods
+from rest_framework import permissions, status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.exceptions import NotFound, ValidationError
+from rest_framework.response import Response
 
+from apartment.models import Project
 from application_form.api.serializers import (
     ApplicationSerializer,
     SalesApplicationSerializer,
 )
+from application_form.exceptions import ProjectDoesNotHaveApplicationsException
 from application_form.models import Application
+from application_form.services.lottery.machine import distribute_apartments
 from audit_log.viewsets import AuditLoggingModelViewSet
 from users.permissions import IsSalesperson
 
 
+@api_view(http_method_names=["POST"])
+@permission_classes([permissions.AllowAny])
```

Comment:
Is this for testing? Since I guess not everyone should be able to do this right?
If it is for testing, please add a TODO so that we don't forget this.

Reply:
All the endpoints which are for the Sales tool now is open endpoints because we don't have authentication implemented. We had a discussion last sprint planning that the authentication will be implemented in January and all these permission checks and others that relate to authentication will be added to JIRA as a ticket that we don't forget these. I made a ticket for this: https://helsinkisolutionoffice.atlassian.net/browse/ASU-1235